code/modules/mob/living/silicon/silicon.dm

```diff
@@ -520,3 +520,6 @@
 
 /mob/living/silicon/get_access()
 	return REGION_ACCESS_ALL_STATION
+
+/mob/living/silicon/reagent_expose(datum/reagent/chem, methods = TOUCH, reac_volume, show_message = TRUE, touch_protection = 0)
+	return COMPONENT_NO_EXPOSE_REAGENTS
```

Comment:
1. did you test this? reading the code, it seems like `/datum/reagent/toxin/zombiepowder/expose_mob` will be called regardless of whatever this returns unless I'm missing something
2. if this does work, does it stop literally all reagent expose procs, like... colorful reagent, as an example? if so, not ideal